CHANGES.md

```diff
@@ -1,3 +1,6 @@
+1.2.0
+* Updated CprSubscription to subscribe and unsubscribe for CPR events.
```

Comment:
Change version not give details of changes in this version, update version details as discussed